app/serializers/answer_serializer.rb

```diff
@@ -0,0 +1,9 @@
+class AnswerSerializer < ActiveModel::Serializer
+  attributes :id
+  attribute :body, key: :name
```

Comment:
Out of curiosity, why rename this from `body` to `name`? I think `body` best describes what this is since its the "body" of the answer.